app/decorators/payment_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PaymentDecorator < Draper::Decorator
+  delegate_all
+
+  def amount
+    amount = read_attribute :amount
+    formatted_amount = "%.2f" % (amount.to_f/100)
+    "#{formatted_amount} €"
```

Comment:
I recommend you to check - [number_to_currency](http://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_to_currency) helper.